script.js

```diff
@@ -1,11 +1,22 @@
 // Create an array called `colors` that contains three colors.
+var colors = [
+  "yellow",
+  "red",
+  "blue"
+]
 
 // Log the last item in the array to the console.
+console.log(colors[2]);
 
 // Create a `for` loop that prints out "My favorite color is [insert color here]" for each item in `colors`.
+for (i=0; i>colors.length; i++)
+  console.log("My favorite color is " + colors[i]);
 
 // Create an `instructor` object that contains three key-value pairs.
+var instructor = {name:"Andy", favoriteFood:"pizza", age:30};
 
 // Add a `facial-hair` property to `instructor`.
+instructor.facial-hair = "mustache";
 
 // Log the value of the `facial-hair` property to the console.
+console.log(instructor.facial-hair);
```

Comment:
![image](https://cloud.githubusercontent.com/assets/17716232/19524530/0bceffc0-95ec-11e6-8f70-e5b1a4259294.png)
You'll get that error, due to the hyphen. You have to use square brackets instead ```js instructor["facial-hair"]="mustache;"```